test/packages/custom_logs/manifest.yml

```diff
@@ -0,0 +1,46 @@
+format_version: 1.0.0
+name: custom_logs
+title: Custom Logs
+description: >-
+  Collect custom logs with Elastic Agent.
+type: input
+version: 1.2.3
+release: ga
+license: basic
+categories:
+  - custom
+policy_templates:
+  - name: first_policy_template
+    type: logs
+    title: Custom log file
+    description: Collect your custom log files.
+    template_path: input.yml.hbs
+    vars:
+      - name: paths
+        type: text
+        title: Paths
+        multi: true
+        required: true
+        show_user: true
+        default:
```

Comment:
I know this is a sample package but I wonder if an input package should even have defaults here?

Reply:
I understand that "defaults" can limit the amount of work on the initial configuration. There might be cases where you don't know what to do and would like to kick off the ingesting pipeline with default settings.

Reply:
I would keep support for `default`, but I wouldn't put any default `paths` in the package for custom logs. I think that we can expect that a user knows at least what logs they want to collect :)

An example of a `default` in a log input package could be to have a variable to support [`ignore_older`](https://www.elastic.co/guide/en/beats/filebeat/8.1/filebeat-input-filestream.html#filebeat-input-filestream-ignore-older), that would default to 0.

Reply:
This is not real `Custom logs` package. I rather wanted to present all options there, which seem to be valid in terms of spec.

Reply:
Well, you can have `paths` as example without defaults, and `ignore_older` as example with defaults :slightly_smiling_face:

Reply:
Fixed!